src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -173,6 +180,12 @@ public enum LowImpedanceBranchMode {
 
     private double plausibleActivePowerLimit = LfNetworkParameters.PLAUSIBLE_ACTIVE_POWER_LIMIT_DEFAULT_VALUE;
 
+    private double maxActivePowerMismatch = MAX_ACTIVE_POWER_MISMATCH;
```

Comment:
NewtonRaphsonStoppingCriteriaType is also a new parameter here, for those 4 new parameters you have to create getters / setters like for all the other parameters to integrate them properly into OpenLoadFlowParameters.